lib/valueset_category_validator.rb

```diff
@@ -112,7 +112,8 @@ def category_appropriate_for_vs(valueset_categories, qrda_oids)
           oid_tuples = @hqmf_qrda_oid_map.find_all {|map_tuple| map_tuple['qrda_oid'] == qrda_oid }
           oid_tuples.each do |oid_tuple|
             categories.each do |category|
-              if oid_tuple['hqmf_name'].include?(CATEGORY_MAP[category])
+              # value set category is ok if it matches the hqmf type, or if vs is an attribute
+              if oid_tuple['hqmf_name'].include?(CATEGORY_MAP[category]) || category == 'Attribute'
                 # if one of the template ids is the right category, true
```

Comment:
This comment and the one you added are both describing line 115 in different ways. Should probably remove one of them to avoid confusion